internal/pkg/otel/translate/status.go

```diff
@@ -267,8 +273,12 @@ func otelStatusToUnitState(status componentstatus.Status) client.UnitState {
 
 // parseEntityStatusId parses an entity status ID into its kind and entity ID. An entity can be a pipeline or otel component.
 // The ID is expected to be in the format "kind:entityId", where kind is either "pipeline" or the otel component type (e.g., "receiver", "exporter").
+// The returned entityId may be empty - this is true for the top-level "extensions" key.
 // This format is used by the healthcheckv2 extension.
 func parseEntityStatusId(id string) (kind string, entityId string) {
+	if id == "extensions" {
+		return "extensions", ""
+	}
```

Comment:
this seems a little bit too specific to `extensions`. I would generalise that in the lines if `id` doesn't contain a `:` then return the `id, ""`, wdyt?

Reply:
Sure, done.